snorkel/slicing/modules/slice_combiner.py

```diff
@@ -54,18 +54,18 @@ def __init__(
         self.temperature = temperature
 
     def forward(  # type:ignore
-        self, flow_dict: Dict[str, List[torch.Tensor]]
+        self, output_dict: Dict[str, torch.Tensor]
```

Comment:
independent from the MultitaskClassifier, it felt a little confusing to name this `output_dict`, given its the input to `forward`

okay for now, but we could consider changing names if something better comes to mind